python/setup.py

```diff
@@ -436,19 +436,23 @@ def copy_libs(dir):
             else:
                 build_prefix = self.build_type
 
+            pyarrow_include = pjoin(build_lib, 'pyarrow', 'include')
+            # Move Arrow C++ headers to pyarrow/include
             if self.bundle_arrow_cpp or self.bundle_arrow_cpp_headers:
                 arrow_cpp_include = pjoin(build_prefix, 'include')
                 print('Bundling includes: ' + arrow_cpp_include)
-                pyarrow_include = pjoin(build_lib, 'pyarrow', 'include')
                 if os.path.exists(pyarrow_include):
                     shutil.rmtree(pyarrow_include)
                 shutil.move(arrow_cpp_include, pyarrow_include)
 
-                # pyarrow/include file is first deleted in the previous step
-                # so we need to add the PyArrow C++ include folder again
-                pyarrow_cpp_include = pjoin(pyarrow_cpp_home, 'include')
-                shutil.move(pjoin(pyarrow_cpp_include, 'arrow', 'python'),
-                            pjoin(pyarrow_include, 'arrow', 'python'))
+            # Move PyArrow headers to pyarrow/include
+            pyarrow_cpp_include = pjoin(pyarrow_cpp_home, 'include')
+            print('Moving PyArrow C++ includes: ' +
+                  pjoin(pyarrow_include, 'arrow', 'python'))
+            if os.path.exists(pjoin(pyarrow_include, 'arrow', 'python')):
+                shutil.rmtree(pjoin(pyarrow_include, 'arrow', 'python'))
+            shutil.move(pjoin(pyarrow_cpp_include, 'arrow', 'python'),
+                        pjoin(pyarrow_include, 'arrow', 'python'))
```

Comment:
Can we define a variable for `pjoin(pyarrow_include, 'arrow', 'python')` because it's used multiple times?